src/Utils.ts

```diff
@@ -17,12 +22,13 @@ export function getTSConfig(modulePath: string): CompilerOptions {
       skipLibCheck: true,
     };
   } else {
-    const tsConfigFile = ts.readConfigFile(tsConfigPath, ts.sys.readFile)
-      .config;
+    const jsonText = await fs.readFile(tsConfigPath);
+    const result = ts.parseJsonText(tsConfigPath, jsonText.toString());
 
-    tsConfigCache = ts.convertCompilerOptionsFromJson(
-      tsConfigFile.compilerOptions,
-      pathDirname(tsConfigPath),
+    tsConfigCache = ts.parseJsonSourceFileConfigFileContent(
+      result,
+      ts.sys,
+      dirname(tsConfigPath),
     ).options;
```

Comment:
This will likely have a small perf impact until I setup proper caching. Need to finish moving this repo over to using @K-FOSS/TS-ESTests and have proper time differences on each PR/commit. Speed is my main priority with TS-ESNode.